src/FPIndia/Site.hs

```diff
@@ -16,12 +16,17 @@ import Optics.Core ((%))
 instance EmaSite Route where
   siteInput cliAct () = do
     staticRouteDyn <- siteInput @StaticRoute cliAct ()
-    markdownDyn <- siteInput @MR.MarkdownRoute cliAct $ def {MR.argBaseDir = "markdown"}
+    markdownDyn <-
+      siteInput @PR.PandocRoute cliAct $
+        def
+          { PR.argBaseDir = "markdown"
+          , PR.argFormats = one ".md"
```

Comment:
Changing this to `PR.argFormats = [".md", ".org"]` means we get org-mode support automatically. 

cc @udaycruise2903